stag-library-compiler/src/main/java/com/vimeo/stag/processor/StagProcessor.java

```diff
@@ -120,15 +128,32 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             }
         }
 
+        Filer filer = processingEnv.getFiler();
         try {
             for (Entry<Element, List<VariableElement>> entry : variableMap.entrySet()) {
                 SupportedTypesModel.getInstance()
                         .addSupportedType(new AnnotatedClass(entry.getKey(), entry.getValue()));
             }
-            ParseGenerator parseGenerator = new ParseGenerator(mSupportedTypes, processingEnv.getFiler());
-            parseGenerator.generateParsingCode();
-            StagGenerator adapterGenerator = new StagGenerator(processingEnv.getFiler());
+            mSupportedTypes.addAll(KnownTypeAdapterFactoriesUtils.loadKnownTypes(processingEnv));
+
+            StagGenerator adapterGenerator = new StagGenerator(filer, mSupportedTypes);
```

Comment:
That's correct.

In my scenario I have an Android application module that includes an Android library, both
of which define entities.  I was a little worried about having the Stag class generated twice
but the Android toolchain seems to do the right thing and use the application module
version.

Ideally I think we'd leverage ServiceLoader from within a non-generated Stag factory class
in the stag-library module.  But...  Android is eating META-INF entries created in the
application module and putting the Stag class under stag-library would require creating
a dependency reference to Gson.  The latter can be handled carefully so as to not force
library versions on folks, etc, but it is still a detractor.